src/components/data-table/column-sorting/column-sort.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Like a java project where each interface is in its own file? I feel like I saw a pattern somewhere with type definitions in their own files, but I can't recall where now. I think it's better practice to separate things out, but it's also more consistent with existing code-base to keep them together.